python/triton/language/core.py

```diff
@@ -535,6 +535,30 @@ def to_ir(self, builder: ir.builder):
 # pointer types
 pi32_t = pointer_type(int32)
 
+
+def get_int_dtype(bitwidth, signed: bool = True) -> dtype:
```

Comment:
Nit: Up to you, but PEP1 "explicit is better than implicit", maybe make `signed` a non-defaulted parameter?

Reply:
Sounds good. Just changed.